src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php

```diff
@@ -104,6 +104,10 @@ public function getListeners($eventName = null)
      */
     public function getListenerPriority($eventName, $listener)
     {
+        if (!method_exists($this->dispatcher, 'getListenerPriority')) {
+            throw new \LogicException(sprintf('The wrapped event dispatcher\'s class ("%s") does not implement the getListenerPriority() method.', get_class($this->dispatcher)));
```

Comment:
another solution would be to fallback to the behavior used by the traceable dispatcher in 2.7 and older: register the listener at priority 0. It is better than breaking the dispatching entirely IMO (especially given that all core implementations have this method implemented so most people will have the right priority anyway)